test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -437,13 +432,11 @@ std::string RandomCode::fillArguments(eth::Instruction _opcode, RandomCodeOption
 			code += getPushCode(randOpMemrGen());	//memlen1
 			break;
 		default:
-			smart = false;
```

Comment:
I think we should throw an exception here. if smart is set but there is no instruction for an opcode here should be an exception